spec/fixtures/job_candidates.json

```diff
@@ -52,7 +50,6 @@
     }
   ],
   "paging": {
-    "next": "https://www.workable.com/spi/v3/accounts/subdomain/jobs/subdomain/candidates?limit=3&since_id=2700d6df"
+    "next": "https://subdomain.workable.com/spi/v3/accounts/subdomain/jobs/subdomain/candidates?limit=3&since_id=2700d6df"
```

Comment:
This doesn't actually match the url specified in the docs at https://workable.readme.io/docs/job-candidates-index